pkg/util/utils.go

```diff
@@ -58,7 +58,7 @@ type DeploymentHelper interface {
 
 // GetManagedBindingLabels returns binding labels for managed cluster.
 func GetManagedBindingLabels(binding *types.SyntheticBinding, managedClusterName string) map[string]string {
-	return map[string]string{constants.K8SAppLabel: constants.VerrazzanoGroup, constants.VerrazzanoBinding: binding.Name, constants.VerrazzanoCluster: managedClusterName}
+	return map[string]string{constants.K8SAppLabel: constants.VerrazzanoGroup, constants.VerrazzanoBinding: binding.Name, constants.VerrazzanoCluster: managedClusterName, constants.IstioInjection: constants.Enabled}
```

Comment:
Does this change end up impacting all namespaces created by this operator, not just the logging namespace?

Reply:
It might, but I'm not aware of any in our current setup.  This change only modified it for the logging namespace in my deployments.  I assumed the only other relevant namespaces would be associated to model binding deployments. Isn't the V-O operator going away and its functionality is being absorbed by other operators?

Reply:
Yes, this operator is eventually going away.  If the logging namespace is the only one it is still creating, then this is fine.